ros_ws/src/crazyswarm/scripts/pycrazyswarm/visualizer/visMatplotlib.py

```diff
@@ -1,14 +1,14 @@
-from mpl_toolkits.mplot3d import Axes3D
+from mpl_toolkits.mplot3d import Axes3D  # monkey patching # NOQA
```

Comment:
Looks great overall - Thanks! What is this changed line for?

Reply:
I have added the first comment (# monkey patching) to avoid the confusion introduced by importing a package that's not used nowhere in the file. In fact both me and my colleague removed this line as the first thing when working with this file in order to later re-introduce it upon realizing it is actually needed. :) If you don't like the brief nature of this comment I could replace it with a sentence describing what this import actually does and why it's needed. 

The second comment (# NOQA) is for linters not to complain that we are importing a module that's not used. I know most of the files in this module are not compliant with PEP8, but maybe it can be slowly improved? In the same fashion spaces after commas were added.

If you prefer to have commits focused on functionality I can include only the things that are needed for the colors to work.

Reply:
Oh, now I've read the question again and realized that maybe I misunderstood it... The line is there to support 3D plots in matplotlib, which are not supported by default. More specifically, it adds the `projection='3D'` option which is used in line 8.

Reply:
I am fine with the NOQA comment, but I would prefer not having "monkey patching" as it stays unclear what it means (to me).
I agree that making the python files pep8 compliant would be very nice. I am fine with merging those minor changes as part of the CL.

If you can remove the "monkey patching" I am happy to merge. Thanks for your contribution!

Reply:
Removed the comment and updated previous commit. Hope it's fine now!

Thanks again for a great work!